vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -71,6 +75,39 @@ def __getitem__(self, arg_name: str):
             return self.type
         return self.figure[arg_name]
 
+    # Interaction methods
+    def _get_figure_interaction_input(self) -> Dict[str, State]:
```

Comment:
Can we make this a `@property` called `_figure_interaction_input`? Similarly in other models.